doc/development/deprecations.rst

```diff
@@ -123,7 +82,48 @@ Other deprecations
 Completed deprecation cycles
 ----------------------------
 
-* The functions ``qml.transforms.sum_expand`` and ``qml.transforms.hamiltonian_expand`` are deprecated.
+* The ``expansion_strategy`` attribute of ``qml.QNode`` is removed.
+  Users should make use of ``qml.workflow.construct_batch``, should they require fine control over the output tape(s).
+
+  - Deprecated in v0.38
+  - Removed in v0.39
+
+* The ``expansion_strategy`` argument in ``qml.specs``, ``qml.draw``, and ``qml.draw_mpl`` is removed. 
+  Instead, use the ``level`` argument which provides a superset of options.
+
+  - Deprecated in v0.38
+  - Removed in v0.39
+
+* The ``max_expansion`` argument in ``qml.QNode`` is removed.
+
+  - Deprecated in v0.38
+  - Removed in v0.39
+
+* The ``expand_fn`` argument in ``qml.execute`` is removed.
+  Instead, please create a ``qml.transforms.core.TransformProgram`` with the desired preprocessing and pass it to the ``transform_program`` argument of ``qml.execute``.
+
+  - Deprecated in v0.38
+  - Removed in v0.39
+
+* The ``max_expansion`` argument in ``qml.execute`` is removed.
+  Instead, please use ``qml.devices.preprocess.decompose`` with the desired expansion level, add it to a ``TransformProgram``, and pass it to the ``transform_program`` argument of ``qml.execute``.
+
+  - Deprecated in v0.38
+  - Removed in v0.39
+
+* The ``override_shots`` argument in ``qml.execute`` is removed.
+  Instead, please add the shots to the ``QuantumTape``\ s to be executed.
```

Comment:
```suggestion
  Instead, please add the shots to the ``QuantumScript``\ s to be executed.
```

It's probably irrelevant, but it seems to me that we are trying to encourage the usage of `QuantumScript` over `QuantumTape`.

Reply:
While that would make me extremely happy, I have admitted defeat in this battle and don't really want to deal with it anymore.